src/lang/fr.json

```diff
@@ -0,0 +1,314 @@
+  "FSC.Notes.Save": "Enregistrer la note",
+  "FSC.Notes.Repeats": "Répétitions",
+  "FSC.Notes.Repeat.Never": "Jamais",
+  "FSC.Notes.Repeat.Monthly": "Mensuel",
+  "FSC.Notes.Repeat.Yearly": "Annuel",
+  "FSC.Notes.Repeat.Weekly": "Hebdomadaire",
+  "FSC.Notes.Time": "Temps",
+  "FSC.Notes.DateTime.Title": "Note Date/Heure",
+  "FSC.Notes.DateTime.Help": "La date de la note et éventuellement l'heure à laquelle la note se produit.",
+  "FSC.Notes.DateTime.AllDay": "Ajouter une heure",
+  "FSC.Notes.Author": "Auteur",
+  "FSC.Date.Sunday": "Dimanche",
+  "FSC.Date.Monday": "Lundi",
+  "FSC.Date.Tuesday": "Mardi",
+  "FSC.Date.Wednesday": "Mercredi",
+  "FSC.Date.Thursday": "Jeudi",
+  "FSC.Date.Friday": "Vendredi",
+  "FSC.Date.Saturday": "Samedi",
+  "FSC.Date.January": "Janvier",
+  "FSC.Date.February": "Février",
+  "FSC.Date.March": "Mars",
+  "FSC.Date.April": "Avril",
+  "FSC.Date.May": "Mai",
+  "FSC.Date.June": "Juin",
+  "FSC.Date.July": "Juillet",
+  "FSC.Date.August": "Août",
+  "FSC.Date.September": "Septembre",
+  "FSC.Date.October": "Octobre",
+  "FSC.Date.November": "Novembre",
+  "FSC.Date.December": "Décembre",
+  "FSC.Time.Current": "Heure actuelle",
+  "FSC.Moon.Phase.New": "Nouvelle Lune",
+  "FSC.Moon.Phase.WaxingCrescent": "Croissante",
+  "FSC.Moon.Phase.FirstQuarter": "Premier trimestre",
+  "FSC.Moon.Phase.WaxingGibbous": "Gibbeuse Croissante",
+  "FSC.Moon.Phase.Full": "Pleine Lune",
+  "FSC.Moon.Phase.WaningGibbous": "Gibbeuse Décroissante",
+  "FSC.Moon.Phase.LastQuarter": "Dernier trimestre",
+  "FSC.Moon.Phase.WaningCrescent": "Décroissante",
+  "FSC.Configuration.Title": "Configuration du calendrier",
+  "FSC.Configuration.Description": "Ce formulaire vous permet de personnaliser votre calendrier de jeux en précisant l'année en cours, le nombre de mois, le nom de chaque mois et le nombre de jours de chaque mois.",
+  "FSC.Configuration.Help": "Pour obtenir de l'aide sur la configuration du calendrier, assurez-vous de consulter la <a href='https://github.com/vigoren/foundryvtt-simple-calendar/tree/main/docs/Configuration.md ' target='_blank'>documentation!</a>",
+  "FSC.Configuration.Save": "Enregistrer la configuration",
+  "FSC.Configuration.General.Title": "Paramètres généraux",
+  "FSC.Configuration.General.PreDefined": "Calendriers prédéfinis",
+  "FSC.Configuration.General.PreDefinedHelp": "Choisissez l'un des calendriers prédéfinis pour démarrer la configuration plus rapidement.",
+  "FSC.Configuration.General.GameWorldTime": "Intégration du temps dans le monde du jeu",
+  "FSC.Configuration.General.GameWorldTimeHelp": "Dites à Simple Calendar comment s'intégrer au temps du monde de jeu et répondre aux mises à jour du temps du monde de jeu.",
+  "FSC.Configuration.General.None": "Aucun : N'est pas lié au temps du monde de jeu.",
+  "FSC.Configuration.General.Self": "Self: Contrôlez le temps du monde de jeu en ignorant les mises à jour des autres modules.",
```

Comment:
This one is hard. An alternative English term I think I could come up with is "Local" instead of "Self" (we could translate using that term instead, which is still "Local" in french :wink:  ), but I am not sure if this is accurate. What do you think @vigoren ?

Reply:
I think local would convey the meaning well enough. This setting is for indicating that only updates from simple calendar are processed so I think local would be close to that meaning.